src/wasm-delegations-fields.def

```diff
@@ -38,6 +38,10 @@
 //    are visited in reverse order, which is convenient for walking by pushing
 //    them to a stack first).
 //
+// DELEGATE_FIELD_REF_CHILD(id, field) - called for each child field whose type
+// is given by an immediate in the binary format. If you do not define this,
+// then DELEGATE_FIELD_CHILD is called.
```

Comment:
Perhaps add "type immediate" to the name here? "ref child" sounds like "child that is a reference type".

Perhaps `DELEGATE_FIELD_BINARY_TYPED_CHILD`?